modules/bitgo/test/v2/unit/coins/utxo/recovery/crossChainRecovery.ts

```diff
@@ -267,17 +187,55 @@ function run<TNumber extends number | bigint = number>(sourceCoin: AbstractUtxoC
       });
     }
 
-    it('should have valid signatures for signed recovery', function () {
-      checkRecoveryTransactionSignature(signedRecovery.txHex as string);
+    it('should test signed cross chain recovery', async () => {
+      const getRecoveryProviderStub = sinon
+        .stub(AbstractUtxoCoin.prototype, 'getRecoveryProvider')
+        .returns(new MockRecoveryProvider<TNumber>(recoveryCoin, getDepositUnspents(), depositTx));
+      const params = {
+        recoveryCoin,
+        txid: depositTx.getId(),
+        recoveryAddress,
+        wallet: recoveryWalletId,
+      };
+      const signedRecovery = (await sourceCoin.recoverFromWrongChain<TNumber>({
+        ...params,
+        xprv: keychainsBase58[0].prv,
+      })) as CrossChainRecoverySigned<TNumber>;
```

Comment:
Due to now stubbing the recovery provider, we cannot make the `signedRecovery` and `unsignedRecovery` in a  before block due to how mocha runs the before blocks (causes conflicts with the stubs and errors out with function is already wrapped)